app/controllers/course_users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class CourseUsersController < ApplicationController
+  before_action :load_course_user, only: :destroy
+
+  def create
+    @course_user = CourseUser.new course_user_params
+    if @course_user.save
+      flash[:success] = t ".add_success"
+      redirect_to course_path(id: params[:course_user][:course_id], join_date: params[:course_user][:join_date] )
+    end
+  end
+
+  def destroy
+    if @course_user.destroy
```

Comment:
Xóa course_user thì subject với task của user sẽ thế nào nhỉ?

Reply:
dạ giờ e mới sửa lại là lúc xóa course_user thì xóa luôn các task and subject của user đó ạ